lib/bridge.js

```diff
@@ -312,6 +312,28 @@ Bridge.prototype._customiseAppservice = function() {
 
         this.addAppServicePath({
             method: "GET",
+            path: "/_matrix/app/:ver/thirdparty/location",
+            handler: function(req, res) {
+                if (req.params.ver !== "unstable" || !lookupController.parseLocation) {
+                    res.status(404).send("Cannot GET " + req.path + "\r\n");
```

Comment:
But it's still not useful, someone might misinterpret that as "GET" is not valid or any part of the path  is invalid. Why not, "version not supported" or "function not supported" (for the ```!lookupController.parseLocation``` bit.